
f = {
    -- This caused an interpreter crash at one point
    1[1] = 1
}

int = make_stack(f)
int.run
assert(int.errored)
